### ISSUE TITLE
Origin 1.3 install failed

### ISSUE THREAD

**Issue by diatmpravin** [openshift/openshift-ansible#2540]
I have working setup of Openshift origin 1.2 cluster with Openstack cloud provider for persistence volume.

I am trying same setup with Openshift origin 1.3 and OpenStack cloud. Below is my Ansible host configuration:

```
[OSEv3:vars]
ansible_ssh_user=root

debug_level=5
deployment_type=origin
openshift_release=v1.3
openshift_image_tag=v1.3.0
openshift_pkg_version=-1.3.0

openshift_router_selector='region=infra'
openshift_registry_selector='region=infra'

```

Error log is:

```

TASK [openshift_common : Install the base package for versioning] **************
fatal: [openshift-master.openshiftpoc.com]: FAILED! => {"changed": false, "failed": true, "msg": "No package origin-1.3.0 available."}

NO MORE HOSTS LEFT *************************************************************
    to retry, use: --limit @/root/openshift-ansible/playbooks/byo/config.retry
```
##### Version

I have checkout to openshift-ansible branch to release 1.3 

```
git describe
openshift-ansible-3.3.27-1-1-gc86c80a

ansible --version
ansible 2.1.1.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
```
##### Additional Information

Fedora 24

**Comment by diatmpravin** [comment]
Am I missing any other setting in OSEv3:vars or specified origin version is wrong? 

Even I can see, origin-1.3.0 is available at [fedora package](http://rpms.famillecollet.com/rpmphp/zoom.php?rpm=origin )

**Comment by diatmpravin** [comment]
Looks rpm package is now available: But I can see in [package list.](http://rpms.famillecollet.com/rpmphp/zoom.php?rpm=origin)

```
yum --showduplicates list origin
Redirecting to '/usr/bin/dnf --showduplicates list origin' (see 'man yum2dnf')

Last metadata expiration check: 2:47:06 ago on Tue Oct  4 06:05:41 2016.
Available Packages
origin.x86_64                                                                               1.1.6-2.git.0.ef1caba.fc24                                                                                fedora 
origin.x86_64                                                                               1.2.0-1.git.0.2e62fab.fc24                                                                                updates
```

 yum install origin
```
Redirecting to '/usr/bin/dnf install origin' (see 'man yum2dnf')

Waiting for process with pid 14183 to finish.
Last metadata expiration check: 2:44:08 ago on Tue Oct  4 06:05:41 2016.
Dependencies resolved.
=============================================================================================================================================================================================================
 Package                                           Arch                                      Version                                                        Repository                                  Size
=============================================================================================================================================================================================================
Installing:
 origin                                            x86_64                                    1.2.0-1.git.0.2e62fab.fc24                                     updates                                     23 M
 origin-clients                                    x86_64                                    1.2.0-1.git.0.2e62fab.fc24                                     updates                                     11 M


```

**Comment by sdodson** [comment]
@diatmpravin It's not available in Fedora 24 as far as I can tell.
@maxamillion Which versions of origin should be available in which versions of Fedora?

**Comment by maxamillion** [comment]
We're targeting to keep the latest version of Origin in all currently supported Fedora releases, however Origin 1.3 is currently failing non-x86_64 platform builds in Fedora 24 because of a golang bug we're currently blocked on. Origin 1.3 is in Rawhide only at the moment.

Current Fedora 24 blocker (because i686 is a Primary Fedora Architecture) https://bugzilla.redhat.com/show_bug.cgi?id=1378960

**Comment by DanyC97** [comment]
cc @vrutkovs candidate for closing as being stale issue

**Comment by vrutkovs** [comment]
Stale issue, please create a new issue if Origin 3.9+ fails to install
